package.json

```diff
@@ -70,7 +70,7 @@
         "express-session": "^1.17.1",
         "express-winston": "^4.0.2",
         "express-ws": "^4.0.0",
-        "formik": "^1.5.8",
+        "formik": "^2.2.5",
```

Comment:
I update formik to 2.x and it seems to work without any code changes from my testing. But would appreciate more testing from reviewers.

Reply:
I tested out app launch (which uses formik the most at the moment) in storybook and in dev mode, and I didn't catch any issues 👍 

I also checked the breaking changes from v1 -> v2, and it doesn't look like any of those will affect us 🎉